src/routes/ThemeProvider.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import { getCurrentWebviewWindow } from "@tauri-apps/api/webviewWindow";
+  import { settings } from "$lib/stores.svelte";
+  import type { FontScale } from "$lib/settings";
+
+  const currentWindow = getCurrentWebviewWindow();
+  const scaleMap: Record<FontScale, string> = {
+    "0": "small",
+    "1": "medium",
+    "2": "large",
+    "3": "x-large"
+  };
+
+  $effect(() => {
+    const documentElement = document.documentElement;
+    const general = settings.app.general;
+
+    if(currentWindow.label === "logs") {
+      const fontSize = scaleMap[general.logScale] ?? "medium";
+      documentElement.style.setProperty("--font-size", fontSize);
```

Comment:
To restore old font scaling, `text-sm` needs to be default, and the chosen font-size set to &lt;html/&gt; for other elements to scale from. But the default font-size is getting overwritten here: https://github.com/snoww/loa-logs/blob/aefa7420e466258cffdb1ef8337c59d90cc3e873/src/app.css#L57-L60

So either restore this div with `text-sm` https://github.com/snoww/loa-logs/blob/bb86d028a67001af281e90ca2a999a6003701668/src/routes/%2Blayout.svelte#L13-L15
Or set `font-size` instead of `--font-size`
```suggestion
      documentElement.style.setProperty("font-size", fontSize);
```

Reply:
would that work?
```
  html {
    font-size: var(--font-size);
  }

  body {
    @apply text-sm text-white;
  }
```